src/main/java/org/example/exceptions/Entity.java

```diff
@@ -0,0 +1,21 @@
+package org.example.exceptions;
+
+public enum Entity {
+    EMPLOYEE("Employee"),
+    SALES_EMPLOYEE("SalesEmployee"),
```

Comment:
Parts of this code isn't needed until later stories, one of the non functional requirements is that each PR contains only the code necessary for the functionality in the story.

Reply:
Thanks, is it ok to comment out? or delete?

Reply:
commented out other entities not needed

Reply:
Lets delete, commented out code doesn't get merged in real projects